test/e2e/subscription_e2e_test.go

```diff
@@ -1300,6 +1300,10 @@ var _ = Describe("Subscription", func() {
 		crClient := newCRClient()
 
 		permissions := deploymentPermissions()
+		permissions2 := deploymentPermissions()
```

Comment:
Is there an issue with shared ServiceAccounts? Why wasn't this an issue in 4.5 and 4.6?